docs/source/API/core/view/view.rst

```diff
@@ -295,31 +322,27 @@ Constructors
 
 .. cppkokkos:function:: View( pointer_type ptr, const IntType& ... indices)
 
-   Unmanaged data wrapping constructor.
+   Unmanaged data-wrapping constructor.
 
-   - ``ptr``: pointer to a user provided memory allocation. Must provide storage of size ``View::required_allocation_size(n0,...,nR)``
+   - ``ptr``: pointer to a user-provided memory allocation. Must provide storage of size ``View::required_allocation_size(n0,...,nR)`` or 
+              ``View::required_allocation_size(layout)``.
```

Comment:
Whats going on here? There is no layout parameter above? Did you try to merge the two constructors??